src/odemis/driver/test/npmc_test.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+'''
+Created on Jun 21, 2018
+
+@author: Anders Muskens
+
+Copyright © 2018 Anders Muskens, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms
+of the GNU General Public License version 2 as published by the Free Software
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+'''
+
+from __future__ import division
+
+import logging
+from odemis.driver import npmc
+import odemis.model as model
+import os
+import time
+import unittest
+
+logging.getLogger().setLevel(logging.DEBUG)
+logging.basicConfig(format="%(asctime)s  %(levelname)-7s %(module)s:%(lineno)d %(message)s")
+
+# Export TEST_NOHW=1 to force using only the simulator and skipping test cases
+# needing real hardware
+TEST_NOHW = (os.environ.get("TEST_NOHW", 0) != 0)  # Default to Hw testing
+
+# arguments used for the creation of basic components
+CONFIG = {"name": "Delay Stage",
+          "role": "delay-stage",
+          "port": "/dev/ttyUSB0",
+          "unit": 's',
+          "axes": {
+                'x': {
+                    'number': 0,
+                    'range': [-1, 1],
+                },
+                'y': {
+                    'number': 1,
+                    'range': [-1, 1],
+                },
+                'z': {
+                    'number': 2,
+                    'range': [-1, 1],
+                },
+           },
+}
+
```

Comment:
Please make a second CONFIG to try the on the real hardware we have (ie, a single axis).

Reply:
Please make this config slightly more complex. At least, use `conv_factor`, and `inverted`. Ideally, it'd have 2 or 3 axes, with different options.